src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.figures;
+
+public abstract class Figure implements Drawable {
+    protected String color;
+    protected double area;
+
+    protected Figure(String color) {
+        this.color = color;
+    }
+
+    public abstract void draw();
+
+    public abstract void calcArea();
```

Comment:
calcArea() need to return double value not void and good practice when you create interface for area calculation